src/lib/pay-request/api_utils/adminUsers.js

```diff
@@ -141,15 +142,38 @@ const adminUsersMethods = function adminUsersMethods(instance) {
     return axiosInstance.patch(path, payload).then(utilExtractData)
   }
 
-  const updateServiceGoLiveStatus = function updateServiceGoLiveStatus(id, status) {
+  const updateServiceDetails = function updateServiceDetails(id, isUpdateStatusToLive, isAlreadyLive, sector, internalFlag,) {
     const path = `v1/api/services/${id}`
-    const payload = {
-      op: 'replace',
-      path: 'current_go_live_stage',
-      value: status
+    const payload = [
+      {
+        op: 'replace',
+        path: 'internal',
+        value: internalFlag
+      },
+      {
+        op: 'replace',
+        path: 'sector',
+        value: sector
+      }
+    ]
+    
+    if (isUpdateStatusToLive && !isAlreadyLive){
```

Comment:
seems like we probably only need one parameter that tells us both things?

Reply:
Done